dev-haskell/cabal-install/cabal-install-3.2.0.0.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.5.9999
+
+CABAL_FEATURES=""
+inherit haskell-cabal
+
+DESCRIPTION="The command-line interface for Cabal and Hackage"
+HOMEPAGE="http://www.haskell.org/cabal/"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+#KEYWORDS="~amd64 ~x86"
+IUSE="debug-conflict-sets debug-expensive-assertions debug-tracetree +native-dns"
```

Comment:
CI is complaining about `debug-tracetree` being an invalid `IUSE`. It probably doesn't exist in the `metadata.xml`, but you may be safe to just hard disable it in `src_configure()`.